### PR TITLE
Tweaks to VNC wrapper

### PR DESCRIPTION
**Pull request by jglick**
@reviewbybees & @olivergondza

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -747,7 +747,27 @@
         </plugins>
       </build>
     </profile>
-    <!-- TODO could be laboriously repeated for $TYPE, $DISPLAY, etc. (seems Maven provides no generic way to do this) -->
+    <profile>
+      <id>BROWSER_DISPLAY</id>
+      <activation>
+        <property>
+          <name>BROWSER_DISPLAY</name>
```
Comment (jglick) [inline pom.xml]: So we can add to `~/.m2/settings.xml` something like

```xml
<profile>
    <id>jenkins-acceptance-test-lts-2</id>
    <properties>
        <JENKINS_WAR>…/.m2/repository/org/jenkins-ci/main/jenkins-war/2.7.2/jenkins-war-2.7.2.war</JENKINS_WAR>
        <LOCAL_SNAPSHOTS>true</LOCAL_SNAPSHOTS>
        <INTERACTIVE>true</INTERACTIVE>
        <RECORDER>off</RECORDER>
        <BROWSER_DISPLAY>:42</BROWSER_DISPLAY>
    </properties>
</profile>
```

File: vnc.sh [hunk 2]
```diff
@@ -7,7 +7,8 @@ if [ ! -z "$1" ]; then
   display=":$1"
 fi
 
-vncserver $display > /dev/null
-vncviewer $display > /dev/null &
+vncserver -kill $display
```
Comment (jglick) [inline vnc.sh]: allows the script to be run repeatedly without manually cleaning up

File: vnc.sh [hunk 3]
```diff
@@ -7,7 +7,8 @@ if [ ! -z "$1" ]; then
...
+vncserver -geometry 1750x1250 $display > /dev/null
```
Comment (jglick) [inline vnc.sh]: default geometry was too small to view Firefox for me

File: vnc.sh [hunk 4]
```diff
@@ -7,7 +7,8 @@ if [ ! -z "$1" ]; then
...
+(vncviewer $display || vncviewer localhost$display) > /dev/null &
```
Comment (jglick) [inline vnc.sh]: `vncviewer-java` does not accept just `:42`, requires `localhost:42`

**Comment by olivergondza** [comment]
:+1:

**Pull request merged**